packages/blade/src/utils/useControllable.ts

```diff
@@ -13,6 +14,11 @@ type ControllableStateSetter<T> = (
    * If `true`, `onChange` won't be called
    */
   skipUpdate?: boolean,
+  /**
+   * Extra data to be passed to `onChange` callback
+   * Example use case: passing event object to `onChange` callback
+   */
+  extraData?: any,
```

Comment:
can we not call this "event" itself?

Reply:
We could, wanted to keep this param a bit generic, imagine in some component maybe not event but we could pass around any data object.